pkg/sriov/sriov.go

```diff
@@ -276,5 +274,27 @@ func (s *sriovManager) ResetVFConfig(conf *types.NetConf) error {
 		}
 	}
 
+	// Reset link guid
+	if conf.HostIFGUID != "" {
+		if conf.HostIFGUID == "00:00:00:00:00:00:00:00" {
+			conf.HostIFGUID = "FF:FF:FF:FF:FF:FF:FF:FF"
+		}
+
+		guid, err := net.ParseMAC(conf.HostIFGUID)
+		if err != nil {
+			return fmt.Errorf("failed to parse guid %s: %v", conf.HostIFGUID, err)
+		}
+		if err = s.nLink.LinkSetVfNodeGUID(pfLink, conf.VFID, guid); err != nil {
+			return fmt.Errorf("failed to add node guid %s: %v", guid, err)
+		}
+		if err = s.nLink.LinkSetVfPortGUID(pfLink, conf.VFID, guid); err != nil {
+			return fmt.Errorf("failed to add port guid %s: %v", guid, err)
+		}
+		// unbind vf then bind it to apply the guid
+		if err = s.utils.RebindVf(conf.Master, conf.DeviceID); err != nil {
+			return err
+		}
```

Comment:
L 283 to 294 look like duplicate code to Line 229

Reply:
add `setVfGUID` function to sriovManger struct to avoid code duplication